python/indices_cff.py

```diff
@@ -107,8 +107,8 @@
 }
 muonRegions = {
     'signal': 'TopPairMuonPlusJetsSelection',
-    'controlRegion1': 'TopPairMuonPlusJetsQCDSelection1p5to3',
-    'controlRegion2': 'TopPairMuonPlusJetsQCDSelection3toInf',
+    'controlRegion1': 'TopPairMuonPlusJetsQCDSelection3toInf',
+    'controlRegion2': 'TopPairMuonPlusJetsQCDSelection1p5to3',
```

Comment:
Just checking these are now in the correct order.  Also in SelectionAnalyzer_cfi etc.